README.md

```diff
@@ -28,3 +28,21 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+## Good Invariant Reasoning:
+My code's invariant is that, at the start of each recursive call to fib(n), the function is designed to return an array that perfectly contains the Fibonacci sequence up to and including the nth element (in alignment with zero-based indexing). Thus ensuring that the 'state of the world' at each step of the recursion accurately builds upon the previous steps, which maintains the integrity of the sequence throughout. <br />
```

Comment:
"...the function is designed to..." -- not sure what you mean. You're not saying anything about what the function does.